### ISSUE TITLE
vue/max-len not work

### ISSUE THREAD

**Issue by UnRealSKY** [idahogurl/vs-code-prettier-eslint#5]
extends
`rvest.vs-code-prettier-eslint@3.0.4`

.eslintrc.js
```
'vue/max-len': [
  'warn',
  {
    code: 140,
    template: 200,
    tabWidth: 2,
    ignoreTemplateLiterals: true,
    ignoreStrings: true,
    ignoreUrls: true,
    ignoreComments: true,
    ignoreHTMLAttributeValues: true,
    ignoreHTMLTextContents: true
  }
]
```

package-lock.json
```
prettier@1.19.1
eslint@6.8.0
prettier-eslint@12.0.0
vue-eslint-parser@7.1.1
```

formater alway follow `max-len` default `code: 80`,
not my setting `vue/max-len` `code: 140` and `template: 200`.

**Comment by idahogurl** [comment]
@UnRealSKY Thank you for creating an issue. I'll get to trying to reproduce it as soon as I can.

**Comment by idahogurl** [comment]
@UnRealSKY Using the example project here: https://github.com/idahogurl/vs-code-prettier-eslint/tree/master/examples/javascript-vue
I was able to get `vue/max-len` to work somewhat where it formatted the line to 100 characters. I used the Prettier-ESLint CLI and got the same result. There is a defect in the Vue ESLint plugin (not mine) that making the ESLint settings not apply.

I ran `eslint *.vue` and received the following:
![image](https://user-images.githubusercontent.com/10620169/126104905-cadf729a-046d-4172-a274-ed0af5a824f8.png)
I used the ESLint configuration you provided but yet it says the max length is 100 not 140.
